docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1149,78 +1149,34 @@
     - key: "kubernetes.azure.com/scalesetpriority"
 </programlisting></para>
 
-        <para>A key is denoted using a minus sign, which is an entry item in
-        the list, which itself is a dictionary with nested attributes. Then
-        the next minus sign (at that same indentation level) is the next entry
-        in that list.</para>
+        <para>A key is denoted using the hyphen, which is an entry item in the
```

Comment:
minor/not new: I think the use of the word "key" here is misleading, there is no key field of a list entry.. there is only an order of entries, the hyphen signifies the start of the next entry in the list.
It's an unfortunate example that the list is a dictionary with the a key field called "key" in the above example..
A list doesn't have to be a list of dictionaries, e.g.:
```
mylist: [ 1, 2, 3 ]
```
which is equivalent to:
```
mylist:
- 1
- 2
- 3
```